src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubOptions.cs

```diff
@@ -37,6 +37,7 @@ public class EventHubOptions : IOptionsFormatter
         public EventHubOptions()
         {
             EventProcessorOptions = EventProcessorOptions.DefaultOptions;
+            PartitionManagerOptions = new PartitionManagerOptions();
```

Comment:
Does this change the default values we were running with before, or do those remain the same?

Reply:
`PartitionManagerOptions` was null for `EventProcessorHost` previosly